src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/Example1.java

```diff
@@ -0,0 +1,32 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AnnotationUseStyle"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.annotation.annotationusestyle;
+
+// xdoc section -- start
+
+@interface SomeArraysUnique {
+  String[] value() default {};
+}
+
+@SuppressWarnings("unchecked") // ok as it's in implied style
```

Comment:
> // xdoc section -- start

please place it above `@SuppressWarnings("unchecked")` in this case example in xdoc file will be same.
this is a comment that define border of what we show to users in html, to let user reed what matters, without all compilation details.

Reply:
done